eclipse-collections-code-generator/src/main/resources/test/map/mutable/abstractMutablePrimitivePrimitiveMapTestCase.stg

```diff
@@ -386,6 +387,29 @@ public abstract class AbstractMutable<name1><name2>MapTestCase extends Abstract<
     }
 
     @Test
+    public void putPair()
+    {
+        Mutable<name1><name2>Map map1 = this.classUnderTest();
+        map1.putPair(PrimitiveTuples.pair(<(literal.(type1))("0")>, <(literal.(type2))("1")>));
+        map1.putPair(PrimitiveTuples.pair(<(literal.(type1))("31")>, <(literal.(type2))("32")>));
+        map1.putPair(PrimitiveTuples.pair(<(literal.(type1))("32")>, <(literal.(type2))("33")>));
+        <name1><name2>HashMap expected = <name1><name2>HashMap.newWithKeysValues(<(literal.(type1))("0")>, <(literal.(type2))("1")>, <(literal.(type1))("31")>, <(literal.(type2))("32")>, <(literal.(type1))("32")>, <(literal.(type2))("33")>);
+        Assert.assertEquals(expected, map1);
+
+        map1.putPair(PrimitiveTuples.pair(<(literal.(type1))("1")>, <(literal.(type2))("2")>));
+        expected.putPair(PrimitiveTuples.pair(<(literal.(type1))("1")>, <(literal.(type2))("2")>));
```

Comment:
I think it would be better if you called put on the expected, since you are testing putPair.  If both putPairs did nothing you would not be testing anything.